recipes/samtools/develop/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set version = "1.9+168_gb1e2c78" %}
+{% set htslib_version = "1.9+353_gefa7aa6" %}
+
+package:
+  name: samtools-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://github.com/samtools/samtools
+  license: MIT
+  summary: C library for high-throughput sequencing data formats.
+
+build:
+  number: 1
```

Comment:
Build number 0?

Reply:
I think I had a 1-based vs 0-based moment - but now people may be testing with the test binary packages I'd rather not drop it to 0 until the version string changes again.